common/persistence/persistenceMetricClients.go

```diff
@@ -910,6 +910,20 @@ func (p *executionPersistenceClient) ReadHistoryBranch(request *ReadHistoryBranc
 	return response, err
 }
 
+func (p *executionPersistenceClient) ReadHistoryBranchReverse(request *ReadHistoryBranchReverseRequest) (
+	*ReadHistoryBranchReverseResponse,
+	error,
+) {
+	p.metricClient.IncCounter(metrics.PersistenceReadHistoryBranchScope, metrics.PersistenceRequests)
```

Comment:
I think it's best to have a separate signal for this API since we expect it to be slower and generally following the pattern of scope per API for ease of debugging/monitoring.